corehq/apps/locations/views.py

```diff
@@ -164,7 +164,10 @@ def mk_loctype(name, parent_type, administrative,
             if loc_type is None:
                 loc_type = LocationType(domain=self.domain)
             loc_type.name = name
-            loc_type.administrative = administrative
+            # only CommTrack domains should have non-administrative locations
+            loc_type.administrative = (administrative
```

Comment:
that sounds potentially clearer. not sure i fully understand how this ties in to the UI. it seems like the right thing to do would be to never show the option to set this whenever commtrack is enabled (but maybe that's already happening in which case your suggestion sounds good)